plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -5101,7 +5119,8 @@ private void createEmptyTable(SchemaTableName schemaTableName, HiveStorageFormat
                     .setTableType(TableType.MANAGED_TABLE.name())
                     .setParameters(ImmutableMap.of(
                             PRESTO_VERSION_NAME, TEST_SERVER_VERSION,
-                            PRESTO_QUERY_ID_NAME, session.getQueryId()))
+                            PRESTO_QUERY_ID_NAME, session.getQueryId(),
+                            TRANSACTIONAL, isTransactional))
```

Comment:
parameter `transactional=false` and lack of `transactional` parameter, are two different situations (which should be handled the same way, but that's another story).
To maintain original test behavior, make `isTransactional` `Optional<String>` and set the table parameter/property only when non-empty.

Or, simpler version, make `isTransactional` a `boolean` and set the param only when true.